scripts/build

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
-/* globals require process */
+/* eslint-env node */
```

Comment:
It's not. We have a  branch underway to normalize the build system, then we'll start implementing that.